src/components/SnapshotDetailView.jsx

```diff
@@ -108,7 +108,15 @@ export class SnapshotDetailView extends React.PureComponent {
   };
 
   render() {
-    const { classes, features, snapshot, snapshotPolicies, terraUrl, canReadPolicies } = this.props;
+    const {
+      classes,
+      features,
+      snapshot,
+      snapshotPolicies,
+      terraUrl,
+      canReadPolicies,
+      dispatch,
+    } = this.props;
     const { filteredDatasets } = this.state;
     const snapshotReadersObj = snapshotPolicies.find((policy) => policy.name === 'reader');
     const snapshotReaders = (snapshotReadersObj && snapshotReadersObj.members) || [];
```

Comment:
There's actually convenience methods for this now in `src/libs/utils.js#getRoleMembersFromPolicies`

Reply:
Nice, sounds like something worth switching to - but, not in this PR.